great_expectations/checkpoint/checkpoint.py

```diff
@@ -110,6 +123,8 @@ class BaseCheckpoint(ConfigPeer):
     instantiated directly (only its descendants, such as "Checkpoint" and "SimpleCheckpoint", should be instantiated).
     """
 
+    DEFAULT_ACTION_LIST: ClassVar[Sequence[Mapping]] = ActionDicts.DEFAULT_ACTION_LIST
```

Comment:
Can we use the `ActionDict` `TypedDict` for all the `Mapping` types?
I forget about it earlier.

https://github.com/great-expectations/great_expectations/blob/f1ebd5ee455c0b47f271d0ac7213ad416040a16f/great_expectations/checkpoint/configurator.py#L30-L41